src/App.js

```diff
@@ -27,7 +35,8 @@ class App extends React.Component{
     return(
       <div className="App">
         <Header />
-        <Main beastData={beastData} showBeast={this.showBeast}/>
+        <HornForm handler={this.handleChange} />
```

Comment:
I would be cautious about naming a prop something different. Just always do it carefully, it can be easy to lose track of props this way in larger programs.

Reply:
Ah, thanks for the advice